### PR TITLE
FoxxiNET

### PR DESCRIPTION
**Pull request by enno123**
## Type of change

 

- [x] Add new directory for a server
- [x] Update directory of a server
- [x] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/foxxinet/manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+   " server_name " : " FoxxiNET " ,
+   " nice_name " : " FoxxiNET " ,
+   " direct_ip " : " Foxxi.net " ,
```
Comment (JakobBruening) [inline minecraft_servers/foxxinet/manifest.json:4]: We prefer lowercase here.

File: minecraft_servers/foxxinet/manifest.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  ],
+  " social " : {
+     " web " : " https://foxxi.net" ,
+     " web_shop " : " https://store.foxxi.net " ,
+     " twitter " : " FoxxiNET " ,
+     " tiktok " : " FoxxiNET " ,
+     " instagram " : " FoxxiNET " ,
+     " discord " : " https://foxxi.net/discord " ,
+     " youtube " : " https://www.youtube.com/channel/UCZCOWPlBtW-y-hQLXT2VYEg " ,
+     " teampeak " : " Foxxi.net "
```
Comment (JakobBruening) [inline minecraft_servers/foxxinet/manifest.json:16]: Wer prefer lowercase here.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
- [ ] Please compress your images https://tinypng.com/.
- [ ] Please format your `manifest.json` as in [our example](https://store.foxxi.net).

And just a suggestion, you can add your branding to the manifest.json (because you have such a beautiful orange as your primary color) - Feel free to do that! :)
=> [Documentation](https://tinypng.com/)

**Review by enno123 — COMMENTED** [review]
~ changed

**Comment by scrummer** [comment]
@enno123 Pay attention: Everything, keys as well as values in your `manifest.json` have whitespaces at the beginning and end. Please remove them according to our examle [manifest.json](https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#example-manifestjson-file)!

BR Scrummer

File: minecraft_servers/foxxinet/manifest.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+   " server_name " : " foxxinet " ,
+   " nice_name " : " foxxinet " ,
+   " direct_ip " : " foxxi.net " ,
+   " server_wildcards " : [
+     "  "
+  ],
```
Comment (scrummer) [inline minecraft_servers/foxxinet/manifest.json:7]: If you do not have any wildcards, please remove this entry.

File: minecraft_servers/foxxinet/manifest.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+     " web_shop " : " https://store.foxxi.net " ,
+     " twitter " : " foxxinet " ,
+     " tiktok " : " foxxinet " ,
+     " instagram " : " foxxinet " ,
+     " discord " : " https://foxxi.net/discord " ,
+     " youtube " : " https://www.youtube.com/channel/UCZCOWPlBtW-y-hQLXT2VYEg " ,
+     " teampeak " : " foxxi.net "
+  },
+  "brand": {
+   "primary": "#db4839",
```
Comment (scrummer) [inline minecraft_servers/foxxinet/manifest.json:19]: Check indentation here (it's one instead of two required spaces)

**Review by scrummer — CHANGES_REQUESTED** [review]

**Comment by enno123** [comment]
- [ ] changed

**Comment by JakobBruening** [comment]
No, there is still a lot broken.
Your json file is invalid. -> https://www.w3schools.com/js/js_json_syntax.asp
Please have a look at JSON syntax.

I think it would be the best to copy-paste our example `manifest.json` file and change or remove the values. So you would have a file with valid syntax, that is also formatted well!

BR, Jakob

**Comment by enno123** [comment]
I'm surprised... I copied the file. I try again

**Comment by enno123** [comment]
I hope it is now correct

**Comment by JakobBruening** [comment]
Nope, my syntax error highlighter is crying 😬



It's not valid json and also, there are waaay to many spaces


Didn't you have have a developer in your team you could ask?

**Comment by enno123** [comment]
I will take care of it :/ Sorry for the inconvenience

**Comment by enno123** [comment]
It should be right now... hopefully

**Pull request merged**